samples/unit-tests/axis/minortickspermajor/demo.js

```diff
@@ -0,0 +1,67 @@
+QUnit.test('Minor ticks per major - linear axis', function (assert) {
```

Comment:
In order to keep processing time down for our test suite, try thinking of how to reduce the number of new charts generated. 

In this case, we can reduce it to one `QUnit.test` and re-use the same chart, just run `chart.yAxis[0].update` in between them to check linear, log and datetime respectively. 

Also consider if we can go even further and piggy back one of the existing tests related to minor ticks.